projects/schematics/src/migrations/2_0/removed-public-api-data.ts

```diff
@@ -237,4 +266,178 @@ export const REMOVED_PUBLIC_API_DATA: DeprecatedNode[] = [
+  },
+  // projects/core/src/state/utils/processes-loader/processes-loader.reducer.ts
+  {
+    node: PROCESSES_LOADER_REDUCER,
+    importPath: SPARTACUS_CORE,
+    comment: `'${PROCESSES_LOADER_REDUCER}' was moved under 'StateUtils' namespace. New reference: 'StateUtils.${PROCESSES_LOADER_REDUCER}'`,
+  },
+  // projects/core/src/state/utils/entity/entity.reducer.ts
+  {
+    node: ENTITY_REDUCER,
+    importPath: SPARTACUS_CORE,
+    comment: `'${ENTITY_REDUCER}' was moved under 'StateUtils' namespace. New reference: 'StateUtils.${ENTITY_REDUCER}'`,
+  },
+  // projects/core/src/state/utils/entity-loader/entity-loader.reducer.ts
+  {
+    node: ENTITY_LOADER_REDUCER,
+    importPath: SPARTACUS_CORE,
+    comment: `'${ENTITY_LOADER_REDUCER}' was moved under 'StateUtils' namespace. New reference: 'StateUtils.${ENTITY_LOADER_REDUCER}'`,
+  },
+  // projects/core/src/state/utils/entity-processes-loader/entity-processes-loader.reducer.ts
+  {
+    node: ENTITY_PROCESSES_LOADER_REDUCER,
+    importPath: SPARTACUS_CORE,
+    comment: `'${ENTITY_PROCESSES_LOADER_REDUCER}' was moved under 'StateUtils' namespace. New reference: 'StateUtils.${ENTITY_PROCESSES_LOADER_REDUCER}'`,
+  },
+  // projects/core/src/state/utils/loader/loader.reducer.ts
+  {
+    node: INITIAL_LOADER_STATE,
+    importPath: SPARTACUS_CORE,
+    comment: `'${INITIAL_LOADER_STATE}' was moved under 'StateUtils' namespace. New reference: 'StateUtils.${INITIAL_LOADER_STATE}'`,
+  },
+  // projects/core/src/state/utils/processes-loader/processes-loader.reducer.ts
+  {
+    node: INITIAL_PROCESSES_STATE,
+    importPath: SPARTACUS_CORE,
+    comment: `'${INITIAL_PROCESSES_STATE}' was moved under 'StateUtils' namespace. New reference: 'StateUtils.${INITIAL_PROCESSES_STATE}'`,
+  },
+  // projects/core/src/state/utils/entity/entity.reducer.ts
+  {
+    node: INITIAL_ENTITY_STATE,
+    importPath: SPARTACUS_CORE,
+    comment: `'${INITIAL_ENTITY_STATE}' was moved under 'StateUtils' namespace. New reference: 'StateUtils.${INITIAL_ENTITY_STATE}'`,
+  },
+  // projects/core/src/state/utils/get-state-slice.ts
+  {
+    node: GET_STATE_SLICE,
+    importPath: SPARTACUS_CORE,
+    comment: `'${GET_STATE_SLICE}' was moved under 'StateUtils' namespace. New reference: 'StateUtils.${GET_STATE_SLICE}'`,
+  },
+  // projects/core/src/state/utils/loader/loader.helpers.ts
+  {
+    node: OF_LOADER_LOAD,
+    importPath: SPARTACUS_CORE,
+    comment: `'${OF_LOADER_LOAD}' was removed.`,
+  },
+  // projects/core/src/state/utils/loader/loader.helpers.ts
+  {
+    node: OF_LOADER_SUCCESS,
+    importPath: SPARTACUS_CORE,
+    comment: `'${OF_LOADER_SUCCESS}' was removed.`,
+  },
+  // projects/core/src/state/utils/loader/loader.helpers.ts
+  {
+    node: OF_LOADER_FAIL,
+    importPath: SPARTACUS_CORE,
+    comment: `'${OF_LOADER_FAIL}' was removed.`,
+  },
+  // projects/core/src/state/utils/entity-loader/entity-loader.selectors.ts
```

Comment:
Those 2 below can be dropped. It won't work as it was previously under a namespace. Info in the migration guide will be enough for those.